microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -518,6 +518,28 @@ def verify_yum_conf(self, node: Node) -> None:
         else:
             raise SkippedException(f"Unsupported distro type : {type(node.os)}")
 
+    @TestCaseMetadata(
+        description="""
+        Verify if there is any issues in and after 'yum update'
+
+        Steps:
+        1. Run 'yum update -y' command.
+        2. Reboot the VM and see if the VM is still in good state.
+        """,
+        priority=2,
+        requirement=simple_requirement(supported_platform_type=[AZURE, READY]),
+    )
+    def verify_yum_update(self, node: Node) -> None:
+        if isinstance(node.os, Fedora):
+            node.execute(
+                "yum update -y",
+                sudo=True,
+                timeout=1800,
+            )
+            node.reboot()
```

Comment:
no need restricts the os, use `node.os.update_packages("")` to invoke the system upgrade

Reply:
Changed the logic to use "node.os.update_packages("")"